src/re_bytes.rs

```diff
@@ -785,6 +785,7 @@ pub struct Captures<'t> {
     named_groups: Arc<HashMap<String, usize>>,
 }
 
+#[allow(len_without_is_empty)]
```

Comment:
It's probably better to

```rust
fn is_empty(&self) -> bool {
    debug_assert!(self.len() > 0);
    false
}
```

as this will trigger const-folding and warnings should anyone expect captures to be empty